engine/modules/opentelemetry/CMakeLists.txt

```diff
@@ -91,7 +91,8 @@ target_link_libraries(opentelemetry
   -L${PROTOBUF_LIB_DIR}
   protobuf
   -L${Boost_LIBRARY_DIR_RELEASE}
-  boost_program_options)
+  boost_program_options
+  protobuf)
```

Comment:
This is strange, isn't it?
you need to link with protobuf twice??

Reply:
rebase issue
fixed